config/example/config.txt

```diff
@@ -17,6 +17,19 @@ JOBS_HAVE_MINIMAL_ACCESS
 ## you have noone older than 0 days, since noone has been logged yet. Only turn this on once you have had the database up for 30 days.
 #USE_AGE_RESTRICTION_FOR_JOBS
 
+##Unhash this to track player playtime in the database. Requires database to be enabled.
+#USE_EXP_TRACKING
+##Unhash this to enable playtime requirements for head jobs.
+#USE_EXP_RESTRICTIONS_HEADS
+##Unhash this to override head jobs' playtime requirements with this number of hours.
+#USE_EXP_RESTRICTIONS_HEADS_HOURS 15
+##Unhash this to change head jobs' playtime requirements so that they're based on department playtime, rather than crew playtime.
+#USE_EXP_RESTRICTIONS_HEADS_DEPARTMENT 
+##Unhash this to enable playtime requirements for certain non-head jobs, like Engineer and Scientist.
+#USE_EXP_RESTRICTIONS_OTHER
+##Unhash this to allow admins to bypass job playtime requirements.
```

Comment:
Options reworked since this comment. The majority of options were removed. This one in particular was left in because it is very useful for testing purposes.